operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -395,22 +395,25 @@ new customized image.
     * Get a comma-separated list of all worker NCN xnames:
 
         ```bash
-        cray hsm state components list --role Management --subrole Worker --type Node --format json |
-          jq -r '.Components | map(.ID) | join(",")'
+        if IFS=$'\n' read -rd '' -a XNAMES; then
+          :
+        fi <<< "$(cray hsm state components list --role Management --subrole Worker --type Node --format json | jq -r '.Components | map(.ID) | join("\n")')"
```

Comment:
As the preceding text indicates, the original command here produced a comma-separated list of worker xnames. The new command produces no output. I believe your new command is intended to be used with the script in the subsequent steps. If so, the text above should probably be adjusted accordingly.

Alternatively, you could just add an additional line to your new command:
```bash
echo "Found the following worker NCNs: ${XNAMES}"
```

That way it would also let them know if the command found no workers, or a weird amount of workers.

Reply:
Yes, it will need rewording.
I changed this because frankly I couldn't believe I was going to have to run these steps umpteen times (since I am not using IUF for this).